pkg/cmd/run/shared/shared.go

```diff
@@ -449,16 +449,25 @@ func PromptForRun(cs *iostreams.ColorScheme, runs []Run) (string, error) {
 	return fmt.Sprintf("%d", runs[selected].ID), nil
 }
 
-func GetRun(client *api.Client, repo ghrepo.Interface, runID string) (*Run, error) {
+func GetRun(client *api.Client, repo ghrepo.Interface, runID string, attempt uint64) (*Run, error) {
 	var result Run
+	var path string
 
-	path := fmt.Sprintf("repos/%s/actions/runs/%s?exclude_pull_requests=true", ghrepo.FullName(repo), runID)
+	if attempt == 0 {
```

Comment:
minor/optional: you could continue setting `path` as before but then modify it if `attempt > 0`. Don't need to do this, it's just usually the pattern I follow.